lib/utils/database.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:path/path.dart';
+import 'package:sqflite/sqflite.dart';
+import 'dart:async';
+
+Future<Database> createDatabase() async {
+  final Future<Database> database = openDatabase(
+    join(await getDatabasesPath(), 'chirpnets.db'),
```

Comment:
Is there anyway to test this?

Reply:
I'll get back to you, probs just gonna add a user and see if it gets added

Reply:
tested :)